### ISSUE TITLE
Reorganize Install page download card to be intuitive

### ISSUE THREAD

**Issue by haoming29** [PelicanPlatform/pelican#905]
Currently we have _five_ options for architectures in the download card: 

![Screenshot 2024-03-06 at 4 51 15 PM](https://github.com/PelicanPlatform/pelican/assets/41393704/c0527f2a-762f-4e55-a433-a0aceb912473)

But `ARM64` and `AARCH64` are the same architecture with different names from debian and RPM. Same thing as `AMD64` VS `X86_64`. According to @matyasselmeci :

> There's no difference; unfortunately Debian packaging uses the "arm64" and "amd64", whereas RPM-based packaging uses "x86_64" and "aarch64"

So we want to merge the two duplicated names, and rename them to be more human-readable and intuitive: ARM, Intel, and PowerPC.

We should be able to safely merge items under `ARM64` and `AARCH64`  together as there's no duplicated binary names, same for `AMD64` VS `X86_64`.

**Comment by CannonLock** [comment]
@Rhymlore Can you take this one please.

**Comment by haoming29** [comment]
@CannonLock @Rhymlore how are we doing here? Do we need to bump this to 7.8?

**Comment by CannonLock** [comment]
@haoming29 

> ARM, Intel, and PowerPC.

Are you okay with use using ARM64, AMD64, and PowerPC? I don't think we should drop the 64 because we don't write any 32 bit software, and I don't think we should use `Intel` because they also make ARM chips so it isn't a good alias for AMD64.

**Comment by CannonLock** [comment]
TLDR for @Rhymlore:

Bin aarch64 and arm64, and amd64 and x86_64 into the corresponding buttons. So when you click ARM, then you will get the aarch64 and arm64 options.

**Comment by haoming29** [comment]
> @haoming29
> 
> > ARM, Intel, and PowerPC.
> 
> Are you okay with use using ARM64, AMD64, and PowerPC? I don't think we should drop the 64 because we don't write any 32 bit software, and I don't think we should use `Intel` because they also make ARM chips so it isn't a good alias for AMD64.

I'm ok with that, but I also want to hear how admins and users react to different chips/architectures. Maybe pin this in `pelican-dev` slack?

**Comment by haoming29** [comment]
This is also part of bigger refactoring as we discussed with RCF last week: #1200
